js/arena.js

```diff
@@ -0,0 +1,88 @@
+import Gladiator from './gladiator.js'
+
+export default class Arena { 
+    constructor (arinaName, gladiators) { 
+this._name=arinaName;
```

Comment:
should be capitalized meaning, 

> Capitalization is the writing of a word with its first letter in uppercase and the remaining letters in lowercase.

the first should be in upper case, while the rest should be in lowercase